internal/server/v2/observation/contained_in.go

```diff
@@ -74,24 +112,46 @@ func FetchChildPlaces(
 	httpClient *http.Client,
 	remoteMixer, ancestor, childType string,
 ) ([]string, error) {
-	childPlacesMap, err := placein.GetPlacesIn(
-		ctx, store, []string{ancestor}, childType)
-	if err != nil {
+	errGroup, errCtx := errgroup.WithContext(ctx)
+
+	storeResponseChan := make(chan map[string][]string, 1)
+	remoteMixerResponseChan := make(chan *pbv2.NodeResponse, 1)
+
+	errGroup.Go(func() error {
+		storeResponse, err := storeFetchChildPlaces(errCtx, store, ancestor, childType)
+		if err != nil {
+			return err
+		}
+		storeResponseChan <- storeResponse
+		return nil
+	})
+
+	if remoteMixer != "" {
+		errGroup.Go(func() error {
+		remoteMixerResponse, err := remoteMixerFetchChildPlaces(metadata, httpClient, ancestor, childType)
+		if err != nil {
+			return err
+		}
+		remoteMixerResponseChan <- remoteMixerResponse
+		return nil
+	})
+	} else {
```

Comment:
i think there is some indent issues. Maybe have a golang plugin to fix them.

Reply:
Done.